Haskell/Haskell.sublime-syntax

```diff
@@ -187,20 +183,25 @@ contexts:
       scope: punctuation.separator.comma.haskell
   block_comment:
     - match: '\{-(?!#)'
-      captures:
-        0: punctuation.definition.comment.haskell
+      scope: punctuation.definition.comment.begin.haskell
       push:
         - meta_scope: comment.block.haskell
+        - match: '\{-#'
```

Comment:
yes, to avoid having to change the comment block match which has a negative lookahead for `#`